views/home.erb

```diff
@@ -3,19 +3,27 @@
   <head>
     <meta charset='UTF-8' />
     <title>Hangman</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa|Inconsolata" rel="stylesheet">
   </head>
-  <body class='container'>
-    <h1> WELCOME TO HANGMAN GAME</h1>
-    <img src='<%= @image %>.jpg' alt='hangman image' height="500" width="500">
-    <p><%= @secret_word %></p>
-    <form action='/play' method='post'>
-      <label for='letter'>Make a guess:</label>
-      <input type='text' id='letter' name='letter_chosen' autofocus="autofocus">
-      <button type='submit' class='btn btn-primary'>submit</button>
-    </form>
-    <p><%= @letters %></p>
-    <p><%= @show_word %></p>
-    <a href='/games-won'>How many times did I win?</a>
+  <body>
+    <div class='container'>
+      <div class='link'>
+	<a href='/games-won'>Win counter</a>
+      </div>
+      <h1><b> Guess the word! </b></h1>
```

Comment:
Keep the spacing consistent. You're not putting spaces around the words inside the HTML tags below, so I would also delete them here. 
This: `<h1><b>Guess the word!</b></h1>`
Not this: `<h1><b> Guess the word! </b></h1>`

Reply:
I was relying on Vim `==` command 😞